data/src/commonMain/kotlin/com/wire/kalium/logic/data/user/type/UserType.kt

```diff
@@ -51,15 +51,44 @@ enum class UserType {
      */
     SERVICE,
 
+    /** Apps, Bots 2.0 **/
+    APP,
+
     /**
      * A user on the same backend,
      * when current user doesn't belongs to any team
      */
     NONE;
 }
 
-fun UserType.isTeammate(): Boolean =
-    this in listOf(UserType.INTERNAL, UserType.ADMIN, UserType.OWNER, UserType.EXTERNAL, UserType.SERVICE)
+sealed class UserTypeInfo(open val type: UserType) {
+    data class Regular(override val type: UserType) : UserTypeInfo(type)
+    data object App : UserTypeInfo(UserType.APP)
+    data object Bot : UserTypeInfo(UserType.SERVICE)
```

Comment:
Maybe it could be `AccountType`?
Where Account can be User, or Bot, App, integration, etc.

And `UserType` can be just to give more information about the User account itself.